Examples/ConstructingPackets/Main.cs

```diff
@@ -36,6 +36,7 @@ public static void Main(string[] args)
 
             // and print out the packet to see that it looks just like we wanted it to
             Console.WriteLine(ethernetPacket.ToString());
+            Console.ReadKey(true);
```

Comment:
I believe so. It happens to be the default setting in VS to spawn another console. Given that this example works fine and is five years old, it's probably best to keep it unchanged, in case somebody decided to grep it in a script or something.